src/libtorch.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
+// Copyright (c) 2019-2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Doesn't this change qualify for our new copyright message?

Reply:
Good point. I'll change it.

Reply:
@dzier can you review the Copyright changes?